build/async.js

```diff
@@ -0,0 +1,1225 @@
+'use strict';
```

Comment:
Yeah, I think I'd prefer to keep them in seperate repos for build size concerns.

I was suggesting `/dist/async.js` for organizational reasons (for instance if we decided to add an `async.async` function in the future it wouldn't be possible due to the organization of built files.